tests/contrib/django/test_django_appsec.py

```diff
@@ -64,6 +65,97 @@ def test_django_request_cookies_attack(client, test_spans, tracer):
         assert query == {"attack": "1' or '1' = '1'"}
 
 
+def test_django_request_body_urlencoded(client, test_spans, tracer):
+    tracer._appsec_enabled = True
```

Comment:
Nit: You have a version of these lines about 12 times:
`    tracer._appsec_enabled = True
    # Hack: need to pass an argument to configure so that the processors are recreated
    tracer.configure(api_version="v0.4")`
    
   You could just add a function like  `_enable_appsec` in test_processor.py (or slightly modify that one and import it), that takes a boolean of whether or not to enable and reuse it throughout so you don't need those 3 lines every time.  
   
  Looks good!

Reply:
Yep. Was already there on other tests so I just cargo cult'ed it. Will do a separate PR for that.